vcs/repository_test.go

```diff
@@ -1152,7 +1175,10 @@ func TestRepository_FileSystem_Symlinks(t *testing.T) {
 			t.Errorf("%s: fs.Lstat(link1): %s", label, err)
 			continue
 		}
-		checkSymlinkFileInfo(label+" (Lstat)", link1Linfo)
+		if runtime.GOOS != "windows" {
+			// TODO(alexsaveliev) make it work
```

Comment:
Yes, this TODO is still valid - at some point we should try fix symlinks functionality on Windows (there are multiple ways with the different outcome of making symlinks: Cygwin's `ln`, MSYS's `ln`, `mklink`) and then make tests run on Windows again